src/scripts/models/content.coffee

```diff
@@ -4,6 +4,7 @@ define (require) ->
   settings = require('settings')
   Collection = require('cs!models/contents/collection')
   Page = require('cs!models/contents/page')
+  $ = require('jquery')
```

Comment:
Thanks for the feedback. It would probably be good to go through all the views and make sure that all dependencies are ordered that way.